Jenkinsfile

```diff
@@ -119,7 +127,10 @@ stage("Check dependent projects against updated schema") {
           [$class: 'StringParameterValue',
             name: 'SCHEMA_BRANCH',
             value: env.BRANCH_NAME],
-        ]
+          [$class: 'StringParameterValue',
+            name: 'SCHEMA_COMMIT',
+            value: env.GIT_COMMIT]
+        ], wait: wait
```

Comment:
Will this make `build` return immediately? If that's the case the statsd timing below won't work anymore.